admin/class-urlslab-admin.php

```diff
@@ -213,4 +214,12 @@ function enqueue_editors_modules( $editor_type ) {
 			}
 		}
 	}
+
+	function admin_body_class( $classes ) {
+		return $this->is_urlslab_admin_page() ? $classes . 'urlslab-admin-page' : $classes;
+	}
+
+	function is_urlslab_admin_page() {
+		return isset( $_GET['page'] ) && str_contains( $_GET['page'], 'urlslab' );
```

Comment:
we should use `sanitize_text_field` for all requests. please change `$_GET['page']` to `sanitize_text_field( $_GET['page'] )`

Reply:
done, also added urlslab body class for add/edit pages as ai assistant will need it because of mui...